Rules.py

```diff
@@ -30,6 +30,14 @@ def set_rules(world):
             if location.name in world.shop_prices:
                 add_item_rule(location, lambda location, item: item.type != 'Shop')
                 location.price = world.shop_prices[location.name]
+                # If price was specified in plando, use it here so access rule is set correctly.
+                if location.name in world.distribution.locations and world.distribution.locations[location.name].price is not None:
+                    price = world.distribution.locations[location.name].price
+                    if price > 999 and price < 32768: # Cap positive values above 999 so that they're not impossible
```

Comment:
Does the comparison that price < 32768 actually do anything?

Reply:
This is a good point. It was important to have that before dotzo's fix with the data size but now it doesn't make as much sense. Basically we have two options:

1) Keep the high value of the range, meaning prices above 32767 will make it to the patching process. This way, the price will be too high and error on the patching process- Which is an informative error, since it'll inform the user the value is too large I believe.

2) Remove the cap, meaning prices too high will just be set to 999, the same way high prices within the range will be. This probably makes more sense now that I think about it.

Let me know what you think- If you agree the 2nd option is better I'll make a quick commit removing the top of the interval.

Reply:
I'd prefer the 2nd option. For one, we should never error in the patching process based on a user input that we could have detected during parsing. But also it doesn't make sense to me to have some range silently capped and some other range cause an error. Either we should throw an error in evaluating the plando for price > 999, or we should just cap to 999. Since it's a simpler fix, I'd say go for capping all positive prices to 999.

Reply:
While I'm at it, I could also prevent prices below -32768, since those will error on patching as well.